ush/python/pygfs/jedi/jedi.py

```diff
@@ -257,6 +261,39 @@ def get_jedi_dict(jedi_config_yaml: str, task_config: AttrDict, expected_block_n
         # Return dictionary of JEDI objects
         return jedi_dict
 
+    @logit(logger)
+    def clean_empty_obsspaces(self):
```

Comment:
This method makes me nervous. Are we 100% sure this doesn't break anything?

Reply:
Ok, I just saw this https://github.com/NOAA-EMC/global-workflow/pull/3645#issuecomment-2898164952 and thought it might be related

Reply:
@CoryMartin-NOAA 
I am about to merge this PR since the test of Gaea passed overnight.  I was just going over comments, is this ok to be resolved?